src/sources/eventstoredb_metrics/mod.rs

```diff
@@ -25,25 +26,29 @@ use crate::{
 pub mod types;
 
 /// Configuration for the `eventstoredb_metrics` source.
+#[serde_as]
 #[configurable_component(source("eventstoredb_metrics"))]
 #[derive(Clone, Debug, Default)]
 pub struct EventStoreDbConfig {
     /// Endpoints to scrape stats from.
```

Comment:
Says endpoints, but the type is a String and example indicates it maybe shouldn't be plural?

Reply:
Good catch!